proposals/jib-cli-surface.md

```diff
@@ -0,0 +1,62 @@
+# Jib CLI Surface Specification
+
+Specification for a command structures exposed to users of the jib-cli
+
+## Usage
+`jib command [OPTIONS]`
+
+## Options
+
+### Required
+```
+-t  --target <image-ref>  a target image reference
+```
+
+### Optional
+```
+-b  --build-file <file>                    location of the build file (default ./jib.yaml)
+    --credHelper <registry>=<credHelper>   credential helper to use for a registry (repeatable)
+    --docker-config <directory>            location of docker configuration
+    --help                                 print usage
```

Comment:
Are these indentations intended?

Reply:
yeah, generally following the format of some command `--help` (like `ls --help`) although I think `java --help` has a slightly different style. Anyway, the indentations are intentional, I can clean it up a little more though